sdk/maps/Azure.Maps.Weather/tests/MapsWeatherTests.cs

```diff
@@ -0,0 +1,308 @@
+            Console.WriteLine("Minimum temperature: " + response.Value.Forecasts[0].Temperature.Minimum.Value);
+            Console.WriteLine("Maximum temperature: " + response.Value.Forecasts[0].Temperature.Maximum.Value);
+            Assert.NotNull(response);
+        }
+
+        [RecordedTest]
+        public async Task GetSevereWeatherAlertsTest()
+        {
+            MapsWeatherClient client = CreateClient();
+            GetSevereWeatherAlertsOptions options = new GetSevereWeatherAlertsOptions()
+            {
+                Coordinates = new GeoPosition(121.5640089, 25.0338053),
+                Language = WeatherLanguage.EnglishUsa
+            };
+            Response<SevereWeatherAlertsResult> response = await client.GetSevereWeatherAlertsAsync(options);
+            if (response.Value.Results.Count > 0) {
+                Console.WriteLine("Description: " + response.Value.Results[0].Description);
+            }
+            Assert.NotNull(response);
+        }
+
+        [RecordedTest]
+        public async Task GetTropicalStormActiveTest()
+        {
+            MapsWeatherClient client = CreateClient();
+            Response<ActiveStormResult> response = await client.GetTropicalStormActiveAsync();
+            Console.WriteLine("Name: " + response.Value.ActiveStorms[0].Name);
+            Assert.NotNull(response);
+        }
+
+        [RecordedTest]
+        public async Task GetTropicalStormForecastTest()
+        {
+            MapsWeatherClient client = CreateClient();
+            GetTropicalStormForecastOptions options = new GetTropicalStormForecastOptions()
+            {
+                Year = 2021,
+                BasinId = BasinId.NP,
+                GovernmentStormId = 2
+            };
+            Response<StormForecastResult> response = await client.GetTropicalStormForecastAsync(options);
+            Console.WriteLine(
+                "Coordinates(longitude, latitude): ("
+                + response.Value.StormForecasts[0].Coordinates.Longitude
+                + ", "
+                + response.Value.StormForecasts[0].Coordinates.Latitude
+                + ")"
+            );
+            Assert.NotNull(response);
+        }
+
+        [RecordedTest]
+        public async Task GetTropicalStormLocationsTest()
+        {
+            MapsWeatherClient client = CreateClient();
+            GetTropicalStormLocationsOptions options = new GetTropicalStormLocationsOptions()
+            {
+                Year = 2021,
+                BasinId = "NP",
+                GovernmentStormId = 2
+            };
+            Response<StormLocationsResult> response = await client.GetTropicalStormLocationsAsync(options);
+            Console.WriteLine(
+                "Coordinates(longitude, latitude): ("
+                + response.Value.StormLocations[0].Coordinates.Longitude
+                + ", "
+                + response.Value.StormLocations[0].Coordinates.Latitude
+                + ")"
```

Comment:
Suggest to use [string interpolation](https://learn.microsoft.com/en-us/dotnet/csharp/tutorials/exploration/interpolated-strings-local) expressions `$"print {varName}"` or [composite formatting](https://learn.microsoft.com/en-us/dotnet/standard/base-types/composite-formatting): `Console.Writeline("{0}, {1}", lat, long)`